deps/wrapper.cc

```diff
@@ -13,6 +13,7 @@ void initialize_coreworker(
     options.language = Language::JULIA;
     options.store_socket = store_socket;    // Required around `CoreWorkerClientPool` creation
     options.raylet_socket = raylet_socket;  // Required by `RayletClient`
+    // XXX: this is hard coded! very bad!!!
```

Comment:
I think there some method `JobID::FromNext()` we can use?

Reply:
or just call it from the cpp init function but that seems a bit wasteful to me somehow.  maybe we don't really need to track state on the julia side though?